src/frontend/css/general.css

```diff
@@ -44,15 +44,82 @@ div.app-container {
   height: max(100vh, 720px);
   width: max(100vw, 1080px);
   display: grid;
-  grid-template-areas: "left right";
+  grid-template-areas: "game board panel";
   grid-template-rows: auto;
-  grid-template-columns: 75% 25%;
+  grid-template-columns: 25% 50% 25%;
   align-items: center;
   justify-items: center;
 }
 
+/* Game Panel */
+
+div.game-panel-container {
+  grid-area: game;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+/*   border: red 1px solid; */
+}
+
+div.game-panel {
+  height: 55%;
+  width: 75%;
+  justify-content: center;
+/*   border: 1px green solid; */
+}
+
+div.game-panel .timer {
+  font-size: 1.4em;
+/*   border: 1px solid blue; */
+}
+
+div.game-panel .player {
+  font-size: 1.2em;
+  line-height: 1.6em;
+/*   border: 1px solid yellow; */
+}
+
+div.game-panel #opp-timer {
+  justify-content: left;
+}
+
+div.game-panel #opp-player {
+  justify-content: left;
+}
```

Comment:
*As-is* these two CSS rules could be merged into `div.game-panel #opp-timer, #opp-player`, but I've kept everything separate for now as we might want to add styles later which fill these rules out and differentiate them.

Reply:
Good idea, I agree